### ISSUE TITLE
[DataCap Application] - iAsk

### ISSUE THREAD

**Issue by wujingenli** [filecoin-project/filecoin-plus-large-datasets#1063]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: iAsk
- Website / Social Media:https://hetongdoc.com/#ishredtid=dpG8Zt&isharejsid=69b00eaa-e300-4067-90e0-b7262691dc31
- Total amount of DataCap being requested (between 500 TiB and 5 PiB):3PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB):60TiB
- On-chain address for first allocation:f1k24fg7bdfsouqoyrfjqvzx4uwezdmlviks2ppaa

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
iAsk is the world's largest Chinese Q&A platform, which was set up in2014 and gathers the wisdom, experience and knowledge of hundreds of millions of netizens, and provides users with a full range of knowledge sharing services such as posting questions, answering questions, searching for answers, data downloads, and entry sharing. Involving people's livelihood, entertainment, IT, food, transportation, shopping and so on.
```

What is the primary source of funding for this project?
business income
```

What other projects/ecosystem stakeholders is this project associated with?
```
nope
```

## Use-case details

Describe the data being stored onto Filecoin
```
The data we want to store on filecoin are contract/agreement/report/PPT/other templates which can be shared with the public. All of them are designed to be used by the public for reference and further practical purpose.
```

Where was the data in this dataset sourced from?
```
accumulative data,which is updated frequently due to the contiuously changing scenarios and customers' needs.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
https://hetongdoc.com/#ishredtid=dpG8Zt&isharejsid=69b00eaa-e300-4067-90e0-b7262691dc31                https://www.sucaiall.com/                                                                                                                                   http://www.bookw.cn/ticai/                                              
https://law.iask.com/converter.html#ishredtid=dwePew&isharejsid=69b00eaa-e300-4067-90e0-b7262691dc31
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
yep
```

What is the expected retrieval frequency for this data?
```
We already opened public service, so it is retrieved everyday.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
Hopefully forever, if not, at least 5 years
```


## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
China
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
online & offline
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
We plan to select 4-5 SPs, which have long-term partenrship
```

How will you be distributing deals across storage providers?
```
as equally as possible, at least 2 copies per SP
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
yep
```

**Comment by wujingenli** [comment]
Hi team, Any feedback on my application? It has been 16 days without response. Look forward to your comments. Thanks!

**Comment by Sunnyiscoming** [comment]
What's the relationship between the organization and http://www.bookw.cn/ticai/, data sample you provided? Have you authorized to store their data as open data on Filecoin network?
Most of data on your website are word. How much data do you have? Can you explain your data composition? How many copies will you store?
What's the relationship between you and the organization?
Can you provide more detailed information about other storage providers participated in this program, such as you can list SPs you have contacted with at present?

**Comment by Sunnyiscoming** [comment]
Close for no reply.

**Comment by wujingenli** [comment]
Dear Sunny

Let me answer your questions.
iAsk has two entities Shenzhen iAsk & Hainan iAsk which were founded in 2012 and 2019 respectively. Shenzhen iAsk invested Hainan iAsk in terms of relationship.
https://www.sucaiall.com/ belongs to Hainan iAsk, and http://www.bookw.cn/ticai/ is parntering with us and put on the bottom of our page for reference. We have talked about this issue and they are glad to do that.

Yes. Most of our data are words, so I applied for 3P. As I stated in the application, I hopefully store them as long as we can. We plan to store 3 copies. 

I am the staff of Shenzhen iAsk.

Since we do domestic business currently, we prefer to select  4 or 5 local SPs first.
